Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/storage/entity/EventCheckInEntity.kt

```diff
@@ -0,0 +1,18 @@
+package de.rki.coronawarnapp.eventregistration.checkins.storage.entity
+
+import android.os.Parcelable
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import de.rki.coronawarnapp.eventregistration.checkins.EventCheckIn
+import kotlinx.parcelize.Parcelize
+import org.joda.time.Instant
+
+@Parcelize
+@Entity(tableName = "checkin")
+data class EventCheckInEntity(
+    @PrimaryKey(autoGenerate = true) @ColumnInfo(name = "id") override val id: Long = 0L,
+    @ColumnInfo(name = "guid") override val guid: String,
+    @ColumnInfo(name = "start") override val start: Instant,
+    @ColumnInfo(name = "end") override val end: Instant
```

Comment:
I think there are more fields according to the latest specs: https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/event-registration-mvp/docs/spec/event-registration-client.md#database-table-for-checkins

Reply:
thank you! added them!